src/Components/EditMovieForm/EditMovieForm.tsx

```diff
@@ -51,12 +51,18 @@ export const EditMovieForm = ({ movie, onSubmit }: EditMovieProps) => {
     onSubmit(formValue);
   }
 
+  function getIdFor(fieldName: string): string {
+    return `${inputIdPrefix}_${fieldName}`;
+  }
+
+  const inputIdPrefix = useId();
+
   return (
     <form className="edit-movie-form" onSubmit={onFormSubmit}>
       <div className="form-fields">
-        <FormField labelTitle="Title: " id="title">
+        <FormField labelTitle="Title: " id={getIdFor('title')}>
```

Comment:
Note: It is not necessary to connect label and input by ids, because you cover input into label the tags will be connected automatically

Reply:
Not sure here.
Input and label tags are located on the same level.
![image](https://user-images.githubusercontent.com/38653807/174869450-20cb3cd5-ec02-454b-be0a-c34e539491c2.png)

Reply:
> Note: It is not necessary to connect label and input by ids, because you cover input into label the tags will be connected automatically

Done.